setup.py

```diff
@@ -14,6 +14,7 @@
     include_package_data=True,
     description='Simple Django app to manage a website social network links',
     long_description=README,
+    long_description_content_type = "text/markdown",
```

Comment:
Please, remove the spaces around the equal sign
```suggestion
    long_description_content_type="text/markdown",
```